app/models/customer.py

```diff
@@ -0,0 +1,23 @@
+from flask import current_app
+from app import db
+from sqlalchemy.orm import relationship
+from sqlalchemy.ext.declarative import declarative_base
+
+class Customer(db.Model):
+    customer_id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    name = db.Column(db.String)
+    postal_code = db.Column(db.String)
+    phone = db.Column(db.String)
+    register_at = db.Column(db.DateTime)
+    videos_checked_out_count = db.Column(db.Integer, default=0)
```

Comment:
Could you pull videos_checked_out_count from the rental table rather than storing it directly in the customer table?